scrapers/sexuria.yml

```diff
@@ -0,0 +1,34 @@
+name: Sexuria
+# movieByURL:
+  # - action: scrapeXPath
+    # url:
+      # - sexuria.net
+    # scraper: sceneScraper
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - sexuria.net
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //*[@id="dle-content"]/article/h1
+      Details:
+        selector: //h4[text()='Description']/following-sibling::div/text()
+        concat: " "
+      Date:
+        selector: //div[text()='Release Date:']/following-sibling::div/text()
```

Comment:
Are you sure you wouldn't rather use `Publication Date` here? I'm not very familiar with Sexuria (it looks like a pirate site?) but every post I checked had a publication date and the ones with release dates were often partial, so they only included a year (which is not valid for local Stash)

Reply:
I am not quite sure with the PublicationDate ( because it is always bigger than the ReleaseDate) so I thought maybe it is the date that the information is published in the site.

Reply:
I didn't know how I could add the default values for missing parts ( like 01-01)

Reply:
I see, you're entirely correct: the publication date has nothing to do with the scene/movie itself 😅